dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/user/AuthenticatedUser.java

```diff
@@ -104,6 +105,9 @@
     @Column(nullable = true)
     private Timestamp lastApiUseTime;   // last API use with user's token
 
+    @Column(nullable = false)
+    private String notificationsLanguage = Locale.ENGLISH.toLanguageTag();
```

Comment:
Can we define this as Locale type field?

Reply:
If we can then i don't know how. Make it `Locale` produces error from entity manager.

Reply:
See: https://github.com/CeON/dataverse/blob/develop/dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/datafile/license/LocaleText.java